torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -279,10 +300,9 @@ def mlir_to_functions(op) -> None:
 
                 scf_results = [Intermediate(idx) for idx in result_ids]
                 for scf_result in scf_results:
-                    for yield_op in yield_ops:
-                        op_stack[parent_block_id][scf_result].append(yield_op)
+                    for return_op in return_ops:
+                        op_stack[parent_block_id][scf_result].append(return_op)
             else:
-                # TODO(oulgen): add support for tt.reduce
                 raise Exception(
```

Comment:
Nit: This should not be a bare exception time, consider runtime exception etc...

Reply:
I'll fix these

Reply:
https://github.com/pytorch/pytorch/pull/121739